docs/examples/export/export-all-maps.md

```diff
@@ -62,6 +63,8 @@ vis_manager = VisualizationManager(auth_client)
 
 # Get all maps from account
 maps = vis_manager.all()
+# initialize progress bar tqdm
+pbar = tqdm(total=len(maps))
```

Comment:
To simplify the tqdm usage, use the [iterable based approach](https://github.com/tqdm/tqdm#iterable-based) so you don't have to manage `pbar`:

```python
for viz in tqdm(maps):
    # your normal code but without pbar 
```